tests/ci/cdk/cdk/codebuild/github_ci_integration_omnibus.yaml

```diff
@@ -74,3 +77,13 @@ batch:
         image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-22.04_gcc-12x_latest
         variables:
           AWS_LC_CI_TARGET: "tests/ci/integration/run_curl_integration.sh"
+
+    - identifier: nginx_integration
+      buildspec: tests/ci/codebuild/common/run_nonroot_target.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_MEDIUM
```

Comment:
What about an ARM build? How long does the overall build and test take?

Reply:
I can give it a shot. The overall build+test takes 10 minutes.

Reply:
Added :)